### PR TITLE
Update steps to elevate and escalate.

### PR DESCRIPTION
**Pull request by mounicabattula**
Making changes to the steps to create MMD elevation and escalation requests after SEP portal migration to MEM. The new changes will be in effect from December 7th 2021 (the release date of SEP portal migration to MEM.)

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/managed-desktop/working-with-managed-desktop/end-user-support.md [hunk 1]
```diff
@@ -41,32 +41,35 @@ Before you request elevated access to a device, it's best to review which action
 
 ### To request elevation
 
-1. Go to the portal at [https://aka.ms/mmdelevationrequest](https://aka.ms/mmdelevationrequest) and sign in with your Azure Active Directory credentials.
-2. Select **New elevation request**.
-3. Provide these details:
+1. Sign in to [Microsoft Endpoint Manager](https://endpoint.microsoft.com/) and navigate to **Devices** menu.
+2. Look for the **Microsoft Managed Desktop** section, and then select **Devices** blade which contains two tabs – Devices Tab and Elevation Requests Tab. 
+3. To create a new elevation request on the Device tab, select a single device that you want to elevate, and then click on **Request elevation** from the Device actions dropdown menu. A new elevation request fly-in pane will appear with the device’s name prepopulated in that field.
+4. Alternatively, to create a new elevation request on the **Elevations Requests tab**, select **+New elevation request.**
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/working-with-managed-desktop/end-user-support.md]:
```suggestion
1. Sign in to [Microsoft Endpoint Manager](https://endpoint.microsoft.com/) and navigate to the **Devices** menu.
2. Look for the **Microsoft Managed Desktop** section, and then select the **Devices** blade, which contains two tabs–the **Devices** tab and the **Elevation requests** tab. 
3. To create a new elevation request on the **Device** tab, select a single device that you want to elevate, and then select **Request elevation** from the Device actions dropdown menu. A new elevation request fly-in pane will appear with the device’s name prepopulated in that field.
4. Alternatively, to create a new elevation request on the **Elevations requests** tab, select **+New elevation request.**
```

File: microsoft-365/managed-desktop/working-with-managed-desktop/end-user-support.md [hunk 2]
```diff
@@ -41,32 +41,35 @@ Before you request elevated access to a device, it's best to review which action
...
+5. Provide these details:
     - **Support ticket ID** from your own support ticketing system.
-    - **Device name**: Enter the device serial number and then select the device from the menu.
+    - **Device name**(only when creating request from **Elevation Requests Tab**): Enter the device serial number and then select the device from the menu.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/working-with-managed-desktop/end-user-support.md]:
```suggestion
    - **Device name**(only when creating request from the **Elevation requests** tab): Enter the device serial number and then select the device from the menu.
```

File: microsoft-365/managed-desktop/working-with-managed-desktop/end-user-support.md [hunk 3]
```diff
@@ -41,32 +41,35 @@ Before you request elevated access to a device, it's best to review which action
...
-1. Go to the portal at [https://aka.ms/mmdelevationrequest](https://aka.ms/mmdelevationrequest) and sign in with your Azure Active Directory credentials.
-2. Select **Escalation requests**, and then select **New escalation request**.
-3. Provide these details:
-    - **Category**: Select the category that best fits your issue.
-    - **Title**: Provide a very brief description.
+1. Sign in to [Microsoft Endpoint Manager](https://endpoint.microsoft.com/) and navigate to the **Tenant administration** menu.
+2. Look for the Microsoft Managed Desktop section, and then select **Service requests**.
+3. On the **Service requests** blade, select **+ New support request**.
+4. Provide a very brief description in the **Title** box. Then set the **Request type** to **Incident**. 
+5. Select the **Category** and **Sub-category** that best fits your issue and click on **Next**.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/working-with-managed-desktop/end-user-support.md]:
```suggestion
5. Select the **Category** and **Sub-category** that best fits your issue and select **Next**.
```

File: microsoft-365/managed-desktop/working-with-managed-desktop/end-user-support.md [hunk 4]
```diff
@@ -41,32 +41,35 @@ Before you request elevated access to a device, it's best to review which action
...
+6. On the Details section, provide the following information:
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/working-with-managed-desktop/end-user-support.md]:
```suggestion
6. In the **Details** section, provide the following information:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @mounicabattula, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@greg-lindsay Please merge this PR. Thanks

**Pull request merged**